.changelog/unreleased/bug-fixes/77-kvindexer-fix-evattribute-indexing.md

```diff
@@ -0,0 +1 @@
+- [state/kvindexer] \#77 Fixed the default behaviour of the kvindexer to index and query attributes by events in which they occur. In 0.34.25 this was mitigated by a separated RPC flag.  (@jmalicevic)
```

Comment:
```suggestion
- `[state/kvindexer]` \#77 Fixed the default behaviour of the kvindexer to index and query attributes by events in which they occur. In 0.34.25 this was mitigated by a separated RPC flag.  (@jmalicevic)
```

Otherwise it's interpreted as a Markdown link.